### PR TITLE
run speedline in gatherer so visual progress is availble to audits

### PR DESCRIPTION
**Pull request by brendankenny**

### CODE REVIEW

**Comment by brendankenny** [comment]
R: @paulirish

**Comment by brendankenny** [comment]
ludicrously long frame lists removed. frame info now only in the json

File: formatters/partials/speedline.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  .speedline-measures {
+    font-size: 14px
+  }
+</style>
+
+<div>
+  <div class="speedline-measures">
+    <div>First Visual Change: <strong>{{this.first}}ms</strong></div>
+    <div>Last Visual Change: <strong>{{this.complete}}ms</strong></div>
+    <div>Recording duration: <strong>{{this.duration}}ms</strong></div>
```
Comment (paulirish) [inline formatters/partials/speedline.html]: we can nuke recording duration from the return value even. doesn't need to be reported.

**Comment by paulirish** [comment]
all lgtm. small nit would be nice to clear up tho

File: formatters/partials/speedline.html (hunk 1 above)
Comment (brendankenny) [inline formatters/partials/speedline.html]: done

**Pull request merged**